cardano-client/src/Cardano/Client/Subscription.hs

```diff
@@ -30,8 +44,9 @@ import qualified Ouroboros.Network.Snocket as Snocket
 
 import qualified Ouroboros.Network.NodeToClient (NodeToClientVersion)
 import           Control.Monad.Class.MonadST (MonadST)
-import           Prelude
 import           Ouroboros.Consensus.Config.SupportsNode (getCodecConfig, getNetworkMagic)
+import           Ouroboros.Network.Mux (MuxPeer (..), RunMiniProtocol (..))
+import           Network.Mux.Trace (MuxTrace, WithMuxBearer)
```

Comment:
Can you please sort the entire import list in this file? This is currently a horrible mix.

Suggestion:

* First all imports from base and standard libs (`Data.Proxy`, etc.)
* Imports from `io-sim-classes` (`Control.Monad.Class.MonadST`)
* Imports from `network-mux` (`Network.Mux.Trace`)
* Imports from `ouroboros-network`
* Imports from `ouroboros-consensus` 

Each of the above sections should be sorted alphabetically and separated by a blank line. Using `stylish-haskell` would also be nice.

Just my 2c :slightly_smiling_face:

Reply:
Thx for the review. I improved the imports. Could you please look at it again.